temp-legacy-code/src/main/java/com/ivy/legacy/legacy/ui/theme/modal/edit/CalculatorModal.kt

```diff
@@ -214,7 +214,8 @@ private fun formatExpression(expression: String, currency: String): String {
 
 private fun calculate(expression: String): Double? {
     return try {
-        Keval.eval(expression.normalizeExpression())
+        val modifiedExpression = if (expression.startsWith("-")) "0$expression" else expression
```

Comment:
I don't understand this `"0$expression"` can you add a comment in the code explaining why we do this?

Reply:
Oh got it, you said it in the PR description but we need this explanation as a comment in the code